### PR TITLE
Change name of NoopMembershipManager

### PR DESCRIPTION
**Pull request by lucyge2022**
### What changes are proposed in this pull request?

Change name of NoopMembershipManager to MasterMembershipManager.

### Why are the changes needed?

N/A

### Does this PR introduce any user facing changes?

N/A

### CODE REVIEW

File: dora/core/common/src/main/java/alluxio/membership/MembershipManager.java [hunk 1]
```diff
@@ -120,7 +120,7 @@ public static MembershipManager create(AlluxioConfiguration conf) {
         case ETCD:
           return EtcdMembershipManager.create(conf);
         case NOOP:
```
Comment (apc999) [inline dora/core/common/src/main/java/alluxio/membership/MembershipManager.java:122]: don't we need to change NOOP to MASTER?
```suggestion
        case MASTER:
```

**Review by apc999 — CHANGES_REQUESTED** [review]
don't we need to change the `enum` ?

File: dora/core/common/src/main/java/alluxio/membership/MembershipManager.java (hunk 1 above)
Comment (lucyge2022) [reply dora/core/common/src/main/java/alluxio/membership/MembershipManager.java]: done

**Comment by lucyge2022** [comment]
> don't we need to change the `enum` ?

thanks for catching, fixed

**Review by apc999 — APPROVED** [review]

**Comment by lucyge2022** [comment]
alluxio-bot, merge this please.

**Pull request merged**